locations/spiders/repont_hu.py

```diff
@@ -1,28 +1,30 @@
-from typing import Iterable
+from typing import Any, Iterable
 
-from scrapy.http import JsonRequest, Request
+import scrapy
+from scrapy.http import JsonRequest, Request, Response
 
 from locations.categories import Categories, Vending, add_vending, apply_category
-from locations.json_blob_spider import JSONBlobSpider
+from locations.dict_parser import DictParser
 
 
-class RepontHUSpider(JSONBlobSpider):
+class RepontHUSpider(scrapy.Spider):
     name = "repont_hu"
     item_attributes = {
         "brand": "REpont",
         "brand_wikidata": "Q130348902",
         "operator": "MOHU MOL Hulladékgazdálkodási Zrt.",
         "operator_wikidata": "Q130207606",
     }
+    start_urls = ["https://map.mohu.hu/api/Map/GetAllPois"]
 
-    def start_requests(self) -> Iterable[Request]:
-        yield JsonRequest(
-            "https://map.mohu.hu/api/Map/SearchWastePoints",
-            data={"wastePointTypes": ["repont"], "hideDrsPoints": False},
-        )
+    def parse(self, response: Response, **kwargs: Any) -> Iterable[Request]:
+        for poi in response.json():
+            yield JsonRequest(
+                f"https://map.mohu.hu/api/Map/GetWastePointDetails?id={poi['id']}", callback=self.parse_details
+            )
 
-    def post_process_item(self, item, response, location):
-        item["name"] = None
+    def parse_details(self, response: Response) -> Any:
+        item = DictParser.parse(response.json())
```

Comment:
These should not have name for sure. I'll see if `located_in` can be mapped.

```suggestion
        item = DictParser.parse(response.json())
        item["name"] = None
```